service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/ErrorConstants.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.adaptors.gp2gp.ehr.utils;
+
+public enum ErrorConstants {
```

Comment:
I would perhaps rename this to `ErrorCode`, as Enums are constants - having `Constants` adds no value.

Here is a [reference](https://stackoverflow.com/a/3069863).